cookbook/email/email.rst

```diff
@@ -138,6 +138,11 @@ template might look something like this:
     {# Makes an absolute URL to the /images/logo.png file #}
     <img src="{{ absolute_url(asset('images/logo.png')) }}"
 
+.. versionadded:: 2.7
+    Support for creating absolute URLs with the ``asset()`` function will
+    be dropped in Symfony 3.0. Starting with Symfony 2.7, you should use
+    the ``absolute_url()`` function instead.
```

Comment:
I updated the text.

I agree with you that this is a bit weird, but imho we should add the note nonetheless to not confuse readers of the docs who actually use older Symfony versions. Though we could think about adding an adapted example for the older maintained versions.